libraries/AP_NavEKF3/AP_NavEKF3.cpp

```diff
@@ -834,6 +836,7 @@ bool NavEKF3::InitialiseFilter(void)
     }
     // exit with failure if any cores could not be setup
     if (!core_setup_success) {
+        num_cores = 0;
```

Comment:
This brokeGPSForYaw.  It's not a flapping test.

Reply:
fixed